guides/gatherers.md

```diff
@@ -98,7 +115,8 @@ Example output (in Rhai):
 };
 ```
 
-### corosync.conf
+<span id="corosyncconf-v1"></span>
```

Comment:
I would go with the `corosyncconfv1` id. This way, `vscode` editor continues working, and github pages/hex docs as well.
So just having the `v1` attached directly to the name
Could you try?

Reply:
yep, works on both editor and docs now :+1: